//package com.sincetimes.redwar.game.comnunicate.protocol;

message RceQueryCheckAndLockAccountIfPossible
{
	repeated HangarsUnitsInfo hangarsUnitsInfo = 1;
	optional int32 planetId = 2;
	optional int32 applyLock = 3;
	optional string targetAccountId = 4;
	optional int32 isAllianceWar = 5;
}

message HangarsUnitsInfo
{
	optional int32 sid = 1;
	optional string sku = 2;
	optional int32 num = 3;
}
